synapse/app/client_reader.py

```diff
@@ -109,12 +112,12 @@ def _listen_http(self, listener_config):
                     ThreepidRestServlet(self).register(resource)
                     KeyQueryServlet(self).register(resource)
                     KeyChangesServlet(self).register(resource)
+                    VoipRestServlet(self).register(resource)
+                    PushRuleRestServlet(self).register(resource)
```

Comment:
surely you can only safely move GET /pushrules to the worker?

Reply:
Oh argh.

Reply:
Fixed?